src/main/java/wolox/training/models/Book.java

```diff
@@ -1,43 +1,45 @@
 package wolox.training.models;
 
-import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.validation.constraints.NotNull;
 
 @Entity
 public class Book {
 
     @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "seq_book")
+    @SequenceGenerator(name = "seq_book", sequenceName = "book_id_seq", allocationSize = 1)
```

Comment:
I think this is not necessary, you can simply use the line `@GeneratedValue(strategy = GenerationType.SEQUENCE)`,  it should suffice  :)